lib/pcb/convert-circuit-json-to-pcb-svg.ts

```diff
@@ -341,6 +342,15 @@ export function convertCircuitJsonToPcbSvg(
     svgObjects = sortSvgObjectsByPcbLayer([...svgObjects, ...ratsNestObjects])
   }
 
+  // Add soldermask layers for copper pours (minimal boolean operations)
+  if (options?.showSolderMask) {
+    const soldermaskObjects = createSvgObjectsForSoldermaskCopperPours(
+      circuitJson,
+      ctx,
+    )
+    svgObjects = sortSvgObjectsByPcbLayer([...svgObjects, ...soldermaskObjects])
```

Comment:
why are we reordering?

Reply:
this code should only sort once, the person who did the rats nest thing made a mistake